pkgs/development/libraries/ndi/version.json

```diff
@@ -1 +1 @@
-{"hash": "sha256:70e04c2e7a629a9854de2727e0f978175b7a4ec6cf4cd9799a22390862f6fa27", "version": "5.5.2"}
+{"hash": "sha256:4ff4b92f2c5f42d234aa7d142e2de7e9b045c72b46ad5149a459d48efd9218de", "version": "5.6.0"}
```

Comment:
Can we just inline this?

Reply:
> Can we just inline this?

I guess it's there because of the python update script.
Should be easy to replace it.

Reply:
Not sure how I feel about this.
The simplicity of the current update script feels like a rather decent solution.

What's the best alternative in your opinion?

Reply:
I don't think that matching & replacing nix code in an update script is nicer than rendering and parsing JSON. This is fine.